server/api/Dockerfile

```diff
@@ -7,6 +7,7 @@ ENV C_FORCE_ROOT true
 ENV HOST 0.0.0.0
 ENV PORT 5001
 ENV DEBUG true
+ENV PATH ~/.planutils/bin:$PATH
```

Comment:
This required in both the api _and_ celery Dockerfile's?

Reply:
I don't believe so, but can add it just in case? I believe the worker is the only one who needs it.

Reply:
Only needed on worker. Made that change.